Services/Models/Proto/GPSPacket.cs

```diff
@@ -0,0 +1,340 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: GPSPacket.proto
+// </auto-generated>
+#pragma warning disable 1591, 0612, 3021
+#region Designer generated code
+
```

Comment:
in the proto file please add:
```
option csharp_namespace = "Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.Protobuf.Models";
```
to avoid risk of collisions when loading classes by name